src/Compilers/CSharp/Portable/Symbols/Extensions/SourceExtensionImplementationMethodSymbol.cs

```diff
@@ -78,10 +81,30 @@ internal sealed override void AddSynthesizedAttributes(PEModuleBuilder moduleBui
                 }
             }
 
-            base.AddSynthesizedAttributes(moduleBuilder, ref attributes);
             SourceMethodSymbol.AddSynthesizedAttributes(this, moduleBuilder, ref attributes);
         }
 
+        internal override void AddSynthesizedReturnTypeAttributes(PEModuleBuilder moduleBuilder, ref ArrayBuilder<CSharpAttributeData> attributes)
+        {
+            if (_originalMethod is SourcePropertyAccessorSymbol { AssociatedSymbol: SourcePropertySymbolBase extensionProperty })
+            {
+                SourcePropertyAccessorSymbol.AddSynthesizedReturnTypeFlowAnalysisAttributes(_originalMethod, extensionProperty, ref attributes);
```

Comment:
>SourcePropertyAccessorSymbol.AddSynthesizedReturnTypeFlowAnalysisAttributes(_originalMethod, extensionProperty, ref attributes); [](http://example.com/codeflow?start=16&length=128)

This is a continuation of the suggestion in SourcePropertyAccessorSymbol.cs:
Here, consider simply calling that instance method on ```_originalMethod``` previously type checked and converted to ```SourcePropertyAccessorSymbol```.